### PR TITLE
Ports a super cool hairstyle for whyfus

### PR DESCRIPTION
**Pull request by Gef21**
New hairstyle ported from somewhere gross, sprites donated by Xsdew and modified by Pawn

What it looks like;
http://i.imgur.com/m63pR9F.png
http://i.imgur.com/pLpFK3N.png
http://i.imgur.com/0Vuxk6O.png
http://i.imgur.com/tiX6xk8.png
http://i.imgur.com/Fse6jNG.png
http://i.imgur.com/r03DViX.png

🆑 
 * rscadd: The hairstyle "Poofy 2" is now available at a mirror near you!

### CODE REVIEW

**Comment by Probe1** [comment]
did you really have to call it that way

**Comment by Gef21** [comment]
call what

**Comment by Probe1** [comment]
Waifu hairstyles

Making my skin crawl here friend

**Comment by Gef21** [comment]
that's what it is though

**Comment by aceedex** [comment]
Call a spade a spade

**Comment by Probe1** [comment]
> poofy2.dmi

Why was this included?

**Comment by Gef21** [comment]
whoops

File: code/modules/mob/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -334,6 +334,11 @@
 		icon_state = "hair_poofy"
 		gender = FEMALE
 
+	poofy2
```
Comment (PJB3005) [inline code/modules/mob/new_player/sprite_accessories.dm:337]: > RELATIVE PATHING

<sup>it's fine</sup>

**Review by PJB3005 — APPROVED** [review]

**Comment by SkowronX** [comment]
lol

File: code/modules/mob/new_player/sprite_accessories.dm (hunk 1 above)
Comment (Probe1) [reply code/modules/mob/new_player/sprite_accessories.dm:337]: I looked at this file once when doing a batch of absolute pathing fixes and concluded they all had to be relatively pathed.

**Pull request merged**